lib/bootloader_setup.pm

```diff
@@ -15,7 +15,7 @@ use Time::HiRes 'sleep';
 use testapi;
 use Utils::Architectures;
 use utils;
-use version_utils qw(is_microos is_sle_micro is_jeos is_leap is_sle is_tumbleweed is_selfinstall is_alp is_transactional);
+use version_utils qw(is_microos is_sle_micro is_jeos is_leap is_sle is_tumbleweed is_slowroll is_selfinstall is_alp is_transactional);
```

Comment:
Now `is_tumbleweed` and `is_slowroll` are not used in this file anymore at all which is great. Please remove both includes. This also discourages other users to incorrectly use those methods.